pyrs/transpiler.py

```diff
@@ -256,6 +261,11 @@ def visit_Attribute(self, node):
 
         value_id = self.visit(node.value)
 
+        if value_id == "sys":
```

Comment:
Move this to pyrs/plugins.py?

Make a new entry into FUNC_DISPATCH_TABLE:

{ sys.argv: visit_sys_argv }

Reply:
It is a common pattern in the visit_Attribute of every transpiler.py , while plugins is pyrs only.

Reply:
Fair enough. We can do this in future diffs which introduce plugins.py to all languages. I was trying to keep py*/transpiler.py as focused as possible and move the API translation to plugins and other files.